samples/CoreAnimationTest/CoreAnimationTest.vsimporter/CoreAnimationTest-WinStore10/CoreAnimationTest-Debug-xcvars.txt

```diff
@@ -87,15 +87,15 @@ PRODUCT_BUNDLE_IDENTIFIER = com.Microsoft.CoreAnimationTest
 PRODUCT_NAME = CoreAnimationTest
 PRODUCT_TYPE = com.apple.product-type.application
 PROJECT = CoreAnimationTest
-PROJECT_DIR = E:/GIT_REPOS/winobjc/samples/CoreAnimationTest
-PROJECT_FILE_PATH = E:\GIT_REPOS\winobjc\samples\CoreAnimationTest\CoreAnimationTest.xcodeproj
+PROJECT_DIR = Y:/Arcadia/WinObjC/samples/CoreAnimationTest
+PROJECT_FILE_PATH = Y:\Arcadia\WinObjC\samples\CoreAnimationTest\CoreAnimationTest.xcodeproj
```

Comment:
that would be awesome. Getting rid of as many of these as possible / converting them into msbuild props would make me ever so happy.

---
In reply to: [102809102](https://github.com/Microsoft/WinObjC/pull/2056#discussion_r102809102) [](ancestors = 102809102)